src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -110,6 +110,36 @@ class MediaChromeSelectMenu extends window.HTMLElement {
     });
   }
 
+  // NOTE: There are definitely some "false positive" cases with multi-key pressing,
+  // but this should be good enough for most use cases.
+  #keyupListener = (e) => {
+    const { key } = e;
+
+    if (!this.keysUsed.includes(key)) {
+      this.removeEventListener('keyup', this.#keyupListener);
+      return;
+    }
+
+    const isButton = e.composedPath().includes(this.#button);
+
+    // only allow Enter/Space on the button itself and not on the listbox
+    // and allow hiding the menu when pressing Escape when focused on the listbox
+    if (isButton && (key === 'Enter' || key === ' ')) {
```

Comment:
if I use space to open the page scrolls down so we need a event.preventDefault() here

Reply:
hm... not happening for me locally, but happy to add one. Has to go in the `keydownListener`, though.

Reply:
updated.